### ISSUE TITLE
Incorrect depth value (should not ignore sign)

### ISSUE THREAD

**Issue by korroktheslavemaster** [wg-perception/linemod#16]
linemod_train.cpp, Line 198:
```
float distance = fabs(renderer_iterator.D_obj() - float(depth.at<ushort>(depth.rows/2.0f, depth.cols/2.0f)/1000.0f));
```
use of `fabs()` is incorrect, since for some objects (eg. a bowl) the distance from (0,0,0) to the object surface along the camera line of sight may be negative. Just removing `fabs()` seems to work.

**Comment by vrabaud** [comment]
@nlyubova , any comment ?

**Comment by nlyubova** [comment]
true about sign, so it is removed in 6e99d3c26536f231d46ef55f61004785fbf0da3d
Thank you!

**Comment by vrabaud** [comment]
ok, fixed in #17

**Comment by vrabaud** [comment]
@korroktheslavemaster , thx for the analysis !
